loleaflet/src/layer/marker/TextInput.js

```diff
@@ -367,7 +395,7 @@ L.TextInput = L.Layer.extend({
 				// TESTME: harder ...
```

Comment:
Quite possibly we could get rid of this comment, and also the similar 'had it been tested' comment in the commit message ;-) I tested with some Chinese speakers, and well - hunting for quick and easy editing outside the BMP plane is/was not easy for me in a hurry. Thanks for mending.